src/Test/AbstractConfigTest.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\DbExtractorConfig\Test;
+
+use Keboola\DbExtractorConfig\Exception\UserException;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Yaml\Yaml;
+
+abstract class AbstractConfigTest extends TestCase
+{
+    public const CONFIG_FORMAT_YAML = 'yaml';
+    public const CONFIG_FORMAT_JSON = 'json';
+
+    /** @var string */
+    protected $dataDir = __DIR__ . '/../../tests/data';
+
+    protected function getConfigDbNode(string $driver): array
```

Comment:
potřebuje to reálné přístupy z envu? Nestačily by prostě statické stringy?

Reply:
já potom tu metodu používám:
https://github.com/keboola/db-extractor-mysql/blob/239073b0387e9138a7fe9080055a1e9cd6a6f813/tests/Keboola/DbExtractor/AbstractMySQLTest.php#L15